.gitignore

```diff
@@ -17,3 +17,6 @@ DartConfiguration.tcl
 doc/Doxyfile
 install_manifest.txt
 
+#buid folder
+build
+
```

Comment:
Not sure this is in line with how GFAL is compiled, up to @mpatrascoiu to approve it (apart the small typo).

Reply:
Hello @glpatcern,

We can leave the build folder in there. Correct the typo, remove the extra trailing whiteline :slightly_smiling_face: